backend/audit/etl.py

```diff
@@ -169,11 +170,10 @@ def load_revision(self):
 
     def load_passthrough(self):
         passthrough = Passthrough(
-            passthrough_name=None,  # TODO: Where does this come from?
-            passthrough_id=None,  # TODO: Where does this come from?
-            audit_id=-1,  # TODO: Where does this come from?
-            audit_year=self.audit_year,
+            award_seq_number=None,  # TODO: Where does this come from?
```

Comment:
This would be award_index. FYI, award_index will be renamed award_reference as per #1513

Reply:
It is currently in `federal_awards` not in `findings_uniform_guidance_entries`. It will be added to `findings_uniform_guidance_entries` as per #1513. In any case, `award_index` will be rename `award_reference`.

Reply:
Where is it in the `FederalAwards` json object?